src/libraries/System.Runtime.Intrinsics/tests/X86/Sse2Tests.ConvertToNUInt.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.Intrinsics.X86;
+using Xunit;
+
+namespace System.Runtime.Intrinsics.Tests.X86;
+
+public sealed partial class Sse2Tests
+{
+    //
+    // Sse2.ConvertToNUInt is currently failing
+    // -> PlatformNotSupportedException
+    // ¯\_(ツ)_/¯
+    //
+    /*
+    [ConditionalTheory(nameof(Run64BitTests))]
+    [InlineData(0, 0, 0)]
+    [InlineData(123, 123, 0)]
+    public void ConvertToNUInt_nuint_64Bit(ulong lower, ulong upper, ulong expected)
+    {
+        Vector128<nuint> value = Vector128.Create(lower, upper).AsNUInt();
```

Comment:
@tannergooding 
Hi Tanner, do you know why this is also failing (PlatformNotSupportedException)?

**intrinsic definition:**
https://github.com/dotnet/runtime/blob/369fdcd5bc0da85fda4b12ae35e03128c2006f10/src/coreclr/jit/hwintrinsiclistxarch.h#L389
https://github.com/dotnet/runtime/blob/369fdcd5bc0da85fda4b12ae35e03128c2006f10/src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/Sse2.cs#L700

**codegen:**
https://github.com/dotnet/runtime/blob/369fdcd5bc0da85fda4b12ae35e03128c2006f10/src/coreclr/jit/hwintrinsiccodegenxarch.cpp#L1290

**lowering:**
https://github.com/dotnet/runtime/blob/369fdcd5bc0da85fda4b12ae35e03128c2006f10/src/coreclr/jit/lowerxarch.cpp#L6041